db/pebble/transaction.go

```diff
@@ -47,6 +47,7 @@ func (t *Transaction) Discard() error {
 // Commit : see db.Transaction.Commit
 func (t *Transaction) Commit() error {
 	start := time.Now()
+	// is it ok that it will count metrics for failed Commit() calls as well?
```

Comment:
That is not ideal, but if we fail to commit a transactio, we have much bigger problems anyways.